docs/data/migration/migration-pickers-v5/migration-pickers-v5.md

```diff
@@ -868,6 +868,17 @@ Component name changes are also reflected in `themeAugmentation`:
  });
 ```
 
+## Stop using `WrapperVariantContext`
+
+The `WrapperVariantContext` context has been removed in favor of more explicit props or correct default values provided by respective picker components.
+This context was only available through internal import, but in case you were using it in your codebase, make sure those cases are updated.
```

Comment:
Maybe we should insist a bit more on the fact that there is nothing to do with this modification

```suggestion
The `WrapperVariantContext` context has been removed.
This context was only available through internal import.

If you did not use this internal context, you should not be impacted by the change.
```

Reply:
We did not document most of the other internal breaking changes, not sure we should document this one.
I often add / remove imports from this file based on what the pro package needs.

Did we recommend someone to use it in an issue ?

Reply:
I do not remember myself recommending it

Reply:
Removed it. 😉